From debe6dd68316e68cdc84f3e638a39fccfca88b42 Mon Sep 17 00:00:00 2001
From: "Craig R. Hughes" <craig.git@rungie.com>
Date: Mon, 7 Nov 2011 17:00:45 -0800
Subject: [PATCH] Minor whitespace fix for clutter color parsing of
 hsla(h,s,l,a)

Previously, if there was whitespace between "l" and the comma before the
alpha value, parsing would fail.  This patch allows that whitespace
making it consistent with whitespace being allowed everywhere else.
---
 clutter/clutter-color.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/clutter/clutter-color.c b/clutter/clutter-color.c
index e8ecf44..17b8b3c 100644
--- a/clutter/clutter-color.c
+++ b/clutter/clutter-color.c
@@ -589,6 +589,7 @@ parse_hsla (ClutterColor *color,
   str += 1;
 
   l = CLAMP (number / 100.0, 0.0, 1.0);
+  skip_whitespace (&str);
 
   /* alpha (optional); since the alpha channel value can only
    * be between 0 and 1 we don't use the parse_rgb_value()
-- 
1.7.7.1

